src/salaryAndBenefits/SalaryAndBenefits.tsx

```diff
@@ -1,17 +1,22 @@
 import styles from "./salaryAndBenefits.module.css";
 import Text from "src/components/text/Text";
 import {SalaryAndBenefits as SalaryAndBenefitsPayload} from "studio/lib/payloads/salaryAndBenefits";
+import Benefit from './components/benefit/Benefit';
 
 interface SalaryAndBenefitsProps {
   salaryAndBenefits: SalaryAndBenefitsPayload
 }
 
 const SalaryAndBenefits = ({salaryAndBenefits}: SalaryAndBenefitsProps) => {
   return (
-    <section className={styles.wrapper}>
-      <Text type={"bodySuperLarge"}>💰❤️</Text>
-      <Text type={"bodySuperLarge"}>{salaryAndBenefits.basicTitle}</Text>
-    </section>
+    <div className={styles.wrapper}>
+      <Text type={"h1"}>{salaryAndBenefits.basicTitle}</Text>
```

Comment:
minor cleanup
```suggestion
      <Text type="h1">{salaryAndBenefits.basicTitle}</Text>
```

Reply:
Common formatting with prettier would be nice 🙃

Reply:
Yes! Let's look at that first thing tomorrow 💡